Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Resources/settings.json

```diff
@@ -0,0 +1,51 @@
+﻿{
+  "platformName": "android",
+  "assemblyNameWithScreens": "Aquality.Appium.Mobile.Template",
```

Comment:
not sure that it is good we have two different names for the same parameters (packageWithScreens  for Java and assembly for C#). Maybe we can use something universal?

Reply:
like what, project name? I believe this would not be possible because of different implementations of reflection functions in these languages

Reply:
@mialeska , @pavelanihimovsky what about "screensPath"?